dnf/cli/commands/__init__.py

```diff
@@ -768,8 +768,9 @@ def __init__(self, cli):
             alias: subcmd for subcmd in subcmd_objs for alias in subcmd.aliases}
 
     def set_argparser(self, parser):
-        super(OptionParser, parser).add_argument('reponame', nargs=1,
-                                                 metavar=_('REPO'))
+        super(OptionParser, parser).add_argument(
+            'reponame', nargs=1, action=OptionParser._RepoCallbackEnable,
```

Comment:
According to man pages:
Repository-Packages Command

The repository-packages command allows the user to run commands on top of all packages in the repository named <repoid>. However, any dependency resolution takes into account packages from all enabled repositories. Specifications <package-name-spec> and <package-spec> further limit the candidates to only those packages matching at least one of them.